src/game_inventory.cpp

```diff
@@ -1614,9 +1622,131 @@ class bionic_install_preset: public inventory_selector_preset
         }
 };
 
-item_location game_menus::inv::install_bionic( player &p, player &patient )
+// Menu used by surgeon when installing a bionic
+class bionic_install_surgeon_preset : public inventory_selector_preset
 {
-    return autodoc_internal( p, patient, bionic_install_preset( p, patient ), 5 );
+    public:
+        bionic_install_surgeon_preset( player &pl, player &patient ) :
+            p( pl ), pa( patient ) {
+            append_cell( [this]( const item_location & loc ) {
+                return get_failure_chance( loc );
+            }, _( "FAILURE CHANCE" ) );
+
+            append_cell( [this]( const item_location & loc ) {
+                return get_operation_duration( loc );
+            }, _( "OPERATION DURATION" ) );
+
+            append_cell( [this]( const item_location & loc ) {
+                return get_money_amount( loc );
+            }, _( "PRICE" ) );
+        }
+
+        bool is_shown( const item_location &loc ) const override {
+            return loc->is_bionic();
+        }
+
+        std::string get_denial( const item_location &loc ) const override {
+            const item *it = loc.get_item();
+            const itype *itemtype = it->type;
+            const bionic_id &bid = itemtype->bionic->id;
+
+            if( it->has_flag( "FILTHY" ) ) {
+                return _( "CBM is filthy." );
+            } else if( !it->has_flag( "STERILE" ) ) {
+                return _( "CBM is not sterile." );
+            } else if( pa.has_bionic( bid ) ) {
+                return _( "CBM already installed" );
+            } else if( bid->upgraded_bionic &&
+                       !pa.has_bionic( bid->upgraded_bionic ) &&
+                       it->is_upgrade() ) {
+                return _( "No base version installed" );
+            } else if( std::any_of( bid->available_upgrades.begin(),
+                                    bid->available_upgrades.end(),
+                                    std::bind( &player::has_bionic, &pa,
+                                               std::placeholders::_1 ) ) ) {
+                return _( "Superior version installed" );
+            } else if( pa.is_npc() && !bid->npc_usable ) {
+                return _( "CBM not compatible with patient" );
+            } else if( loc.get_item()->price( true ) * 2 > p.cash ) {
+                return string_format( _( "%s" ), format_money( loc.get_item()->price( true ) * 2 ) );
```

Comment:
```suggestion
                return format_money( loc.get_item()->price( true ) * 2 );
```
Translating the string "%s" is pointless, the translators don't know the context and can't really add anything. And a call to `string_format` with the string "%s" is also pointless as it will just forward the input string.